resharper/resharper-unity/src/Application/UI/Help/ShowUnityHelp.cs

```diff
@@ -37,25 +36,39 @@ public bool ShowHelp(string keyword, HelpSystem.HelpKind kind, string preferredP
         public double Priority => 0;
         public HelpSystem.HelpKind[] SupportedKinds => new[] {HelpSystem.HelpKind.Msdn};
 
-        private static bool IsUnityKeyword(string keyword)
+        public static bool IsUnityKeyword(string keyword)
         {
-            return keyword.StartsWith("UnityEngine.", StringComparison.OrdinalIgnoreCase)
-                   || keyword.StartsWith("UnityEditor.", StringComparison.OrdinalIgnoreCase);
+            return keyword !=null && 
+                   (keyword.StartsWith("UnityEngine.", StringComparison.OrdinalIgnoreCase)
+                   || keyword.StartsWith("UnityEditor.", StringComparison.OrdinalIgnoreCase));
         }
 
-        private static string StripPrefix(string keyword)
+        public static string StripPrefix(string keyword)
```

Comment:
This shouldn't be public. It makes a huge assumption about the value of the text, so we should only call it from this context.